pkg/virt-controller/watch/migration.go

```diff
@@ -248,41 +247,33 @@ func ensureSelectorLabelPresent(migration *virtv1.VirtualMachineInstanceMigratio
 }
 
 func (c *MigrationController) patchVMI(origVMI, newVMI *virtv1.VirtualMachineInstance) error {
-	var ops []string
+	fmt.Printf("XXX patchVMI\n")
```

Comment:
`Printf`

Reply:
... and some more down below

Reply:
Yes, this is a leftover. You were too quick with the review. I should have written more explicitly that this is WIP